app/src/main/res/menu/menu_talk.xml

```diff
@@ -4,22 +4,39 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="AlwaysShowAction">
-    <item android:id="@+id/menu_change_language"
+    <item
+        android:id="@+id/menu_change_language"
         android:icon="@drawable/ic_translate_themed_24dp"
         android:title="@string/langlinks_activity_title"
         app:showAsAction="always" />
-    <item android:id="@+id/menu_notifications"
+    <item
+        android:id="@+id/menu_notifications"
         android:title="@string/notifications_activity_title"
-        app:showAsAction="always"/>
-    <item android:id="@+id/menu_view_user_page"
-        android:title="@string/go_to_user_page"
-        app:showAsAction="never" />
-    <item android:id="@+id/menu_view_in_browser"
-        android:title="@string/page_view_in_browser"
-        app:showAsAction="never" />
+        app:showAsAction="always" />
+
+
     <item
-        android:id="@+id/menu_talk_topic_share"
-        android:icon="@drawable/ic_share_material_secondary_themed"
-        android:title="@string/menu_option_share_talk_page"
-        app:showAsAction="never" />
+        android:id="@+id/menu_notifications_group"
+        android:icon="@drawable/ic_more_vert_themed_24dp"
```

Comment:
Looks like the `themed` icons are using different color attributes here. 

The `ic_translate_themed_24dp` is using `material_theme_secondary_color` but the notifications and more menu are using `toolbar_icon_color`. Could you please check which one is the correct one?